### PR TITLE
Rename packages (nyc -> coverage, prettier -> format, tsc -> typescript)

### LINKED ISSUE #22: Coveralls reporting not working
Discovered during a check on an ``nyc`` version update PR on the blockchain library that coveralls reporting is currently not working when using the scripts from the config library:
https://github.com/ethereumjs/ethereumjs-blockchain/pull/97

Did some tests and can confirm that removing the all ``exec`` statements from both the ``coverage`` and the ``coveralls`` command fixes this.

### PR DESCRIPTION
**Pull request by ryanio**
As per https://github.com/ethereumjs/ethereumjs-config/pull/26#issuecomment-588427106 and to close https://github.com/ethereumjs/ethereumjs-config/issues/18, this PR renames:

* `ethereumjs-config-nyc` -> `ethereumjs-config-coverage`
* `ethereumjs-config-prettier` -> `ethereumjs-config-format`
* `ethereumjs-config-tsc` -> `ethereumjs-config-ts-compile`
`ethereumjs-config-build` -> `ethereumjs-config-ts-build`

`ethereumjs-config-tslint` is renamed to `ethereumjs-config-lint` in #26.

This PR also removes coveralls since we can now push reports via github actions (closes #22).

### CODE REVIEW

File: packages/typescript/README.md [hunk 1]
```diff
@@ -38,8 +38,8 @@ Use CLI commands above in `package.json`:
 
 ```json
   "scripts": {
-    "tsc": "ethereumjs-config-tsc",
-    "build": "ethereumjs-config-build"
+    "tsc":   "ethereumjs-config-ts-compile",
+    "build": "ethereumjs-config-ts-build"
```
Comment (ryanio) [inline packages/typescript/README.md:42]: great!

**Comment by ryanio** [comment]
thanks for those, looks great!

**Comment by evertonfraga** [comment]
I looked at past PRs, and it seems we can safely ignore LGTM analysis for JS. Merging in…

**Comment by holgerd77** [comment]
👍

**Pull request merged**